crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -675,17 +675,30 @@ impl ReadOnlyAccount {
         self.inner.lock().await.sign(string)
     }
 
-    /// Check that the given json value is signed by this account.
+    /// Check if the given JSON is signed by this Account key.
+    ///
+    /// This method should only be used if a signature of an object should be
+    /// checked multiple times and the canonicalization step wants to be done
+    /// only a single time.
```

Comment:
```suggestion
    /// This method should only be used if an object's signature needs to be
    /// checked multiple times, and you'd like to avoid performing the
    /// canonicalization step each time.
```